pyo3-build-config/src/impl_.rs

```diff
@@ -441,6 +441,24 @@ print("mingw", get_platform().startswith("mingw"))
         let implementation = implementation.unwrap_or(PythonImplementation::CPython);
         let abi3 = abi3.unwrap_or(false);
 
+        // Auto generate python3.dll import libraries for Windows targets.
+        #[cfg(feature = "python3-dll-a")]
```

Comment:
I'm not sure whether this is the right place to do this, feel free to suggest a better approach.

Reply:
Perhaps a new `InterpreterConfig` method like `fixup_import_libs(&mut self)` would be more readable and easier to feature-gate.

You can then invoke it explicitly after `from_reader()` here:

https://github.com/PyO3/pyo3/blob/87bd10c9a33e0a189248f32e46fd1c602ab5606a/pyo3-build-config/src/lib.rs#L179-L180